iio/iio.c

```diff
@@ -1345,11 +1486,32 @@ static int32_t iio_init_devs(struct iio_desc *desc,
 		}
 	}
 
-	ret = iio_init_xml(desc);
-	if (IS_ERR_VALUE(ret))
-		free(desc->devs);
+	return SUCCESS;
+}
 
-	return ret;
+static int32_t iio_init_trigs(struct iio_desc *desc,
+			      struct iio_trigger_init *trigs, int32_t n)
+{
+	uint32_t i;
+	struct iio_trig_priv *ltrig;
+	struct iio_trigger_init *ntrig;
```

Comment:
I'd rename `ltrig` and `ntrig` to something more explicit and less likely to be confused with one-another. `ntrig` could be `init_iter` and `ltrig` could be `priv_iter`.